src/views/shared/client/utils.tsx

```diff
@@ -86,18 +87,36 @@ export function ReplaceNewlines(text: string): string {
     .join("");
 }
 
-export function LoadCardImageIntoElement(
+export async function LoadCardImageIntoElement(
   cardId: string,
   dataDetails: DataDetailsResponse,
   element: HTMLElement
-): void {
+): Promise<void> {
   if (element.getAttribute("data-loaded")) {
     return;
   }
   element.setAttribute("data-loaded", "true");
+  element.style.backgroundImage = "url(/Images/CardBack.jpg)";
   const imageUrl = GetImageUrl(cardId, dataDetails);
-  // eslint-disable-next-line prettier/prettier
-  element.style.backgroundImage = `url("${imageUrl}")`;
+
+  const result = await resolveOrTimeout(waitForImageToLoad(imageUrl), 100);
```

Comment:
`result` is kind of generic, that name is usually used when a function only does 1 thing or calls 1 helper function. Here, there are multiple things going on so I think this warrants a more specific name. Maybe `timeoutStatus` or something?